package.json

```diff
@@ -130,8 +130,8 @@
     "lint-staged": "12.2.0",
     "prettier": "2.5.1",
     "serve": "^13.0.2",
-    "storybook-addon-gatsby": "^0.0.3",
-    "storybook-react-i18next": "^1.0.16",
+    "storybook-addon-gatsby": "0.0.3",
```

Comment:
Why did you decide to pin this version down? Is there something wrong with `0.0.4` or `0.1.0`?